libgcc/config/riscv/multi3.S

```diff
@@ -79,3 +80,4 @@ __multi3:
   mv  a0, t2
   mv  a1, t4
   jr  t0
+#endif
```

Comment:
I suggest that for RV32E, we rewrite this in C code, and inline the call to __mulsi3, so that RA doesn't need to be spilled.